cpp/trtorchc/main.cpp

```diff
@@ -230,7 +230,7 @@ int main(int argc, char** argv) {
   }
 
   if (verbose) {
-    trtorch::logging::set_reportable_log_level(trtorch::logging::Level::kDEBUG);
+    trtorch::logging::set_reportable_log_level(trtorch::logging::Level::kGRAPH);
```

Comment:
We can revert this back as kGRAPH prints too much info.

Reply:
Thanks, I did not intend to include this debug change in PR.